styles/constants.scss

```diff
@@ -0,0 +1,42 @@
+/*
+    Constants
```

Comment:
are these necessary? won't these override the existing styles of other components?

Reply:
the values `FONT_STACK` and `BASE_FONT_SIZE` are not being used so ill delete them, but the others are purely only used in the new Calendar component.